ibm_catalog.json

```diff
@@ -0,0 +1,325 @@
+{
+    "products": [
+        {
+            "name": "deploy-arch-ibm-ocp-virtualization",
+            "label": "Cloud automation for Red Hat OpenShift Virtualization",
+            "product_kind": "solution",
+            "tags": [
+                "containers",
+                "target_terraform",
+                "target_roks",
+                "terraform",
+                "solution",
+                "ibm_created",
+                "ibm_beta"
+            ],
+            "keywords": [
+                "openshift",
+                "openshift-virtualization",
+                "ocp-virtualization",
+                "virtualization",
+                "cluster",
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution"
+            ],
+            "short_description": "Rapidly deploy Red Hat OpenShift clusters on Virtual Private Cloud (VPC) pre-configured with OpenShift Virtualization capabilities for VM workloads.",
+            "long_description": "Red Hat® OpenShift® Virtualization, provides a modern platform for organizations to run and deploy their new and existing virtual machine (VM) workloads. This automation streamlines the deployment of Red Hat OpenShift cluster with OpenShift Virtualization on IBM Cloud. The OpenShift Virtualization Operator adds Kubernetes custom resources to enable virtualization.\n\nℹ️ This Terraform-based automation is part of a broader suite of IBM-maintained Infrastructure as Code (IaC) assets, each following the naming pattern \"Cloud automation for *servicename*\" and focusing on single IBM Cloud service. These single-service deployable architectures can be used on their own to streamline and automate service deployments through an [IaC approach](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-understanding-projects), or assembled together into a broader [automated IaC stack](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-config-stack) to automate the deployment of an end-to-end solution architecture.",
+            "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-ocp-virtualization/blob/main/README.md",
+            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-virtualization/main/images/rh_ocp.svg",
+            "provider_name": "IBM",
+            "features": [
+                {
+                    "title": "OpenShift Virtualization",
+                    "description": "Automates the provisioning of a fully configured Red Hat OpenShift Cluster on VPC with OpenShift Virtualization capabilities on IBM Cloud. The OpenShift Virtualization Operator adds Kubernetes custom resources to enable virtualization. [Learn more](https://www.ibm.com/docs/en/fusion-software/2.8.x?topic=red-hat-openshift-virtualization)"
```

Comment:
OpenShift cluster is hard dependency. So we can add OpenShift cluster and some related features.